packages/drivers/odsp-driver-definitions/package.json

```diff
@@ -95,7 +95,7 @@
 		"@fluidframework/build-common": "^2.0.3",
 		"@fluidframework/build-tools": "^0.40.0",
 		"@fluidframework/eslint-config-fluid": "^5.3.0",
-		"@fluidframework/odsp-driver-definitions-previous": "npm:@fluidframework/odsp-driver-definitions@2.0.0-rc.5.0.0",
+		"@fluidframework/odsp-driver-definitions-previous": "npm:@fluidframework/odsp-driver-definitions@2.0.0",
```

Comment:
Noting for follow up - find the commit which would have added the "broken" entries.